README.md

```diff
@@ -18,6 +18,9 @@ After this command you can open the project by opening or running on a live serv
 ### Run webpack dev server
 `npm run build:server`
 
+### CSS Configuration
+`npm run build:css` This command reads the `tailwind.config.js` file and `styles.css` template file, transpiles the CSS and exports a `styles.css` file to the `build` folder.
```

Comment:
Remember to create your branchs always from dev uptated from origin. This change should not be in here!!

Reply:
In general, you have changes from the tailwind task in here. Need to clean this app to avoid problems.

Reply:
Ok, since I updated the dev branch with the config-tailwind branch, when I update dev it will have these changes. So when I merge my config-eslint it will have these changes. But for the future I will make sure to branch from the most updated dev branch and not off of any working/feature branches.